prompb/types.proto

```diff
@@ -127,6 +147,7 @@ message TimeSeries {
   repeated Sample samples       = 2 [(gogoproto.nullable) = false];
   repeated Exemplar exemplars   = 3 [(gogoproto.nullable) = false];
   repeated Histogram histograms = 4 [(gogoproto.nullable) = false];
+  repeated Metadata metadatas   = 5 [(gogoproto.nullable) = false];
```

Comment:
👋🏻, naive question why is this a repeated field?

Reply:
I guess semantically it matches the other fields which support multiple samples under the same series. I just don't understand in what cases we'd have different metadata for the same series.

Reply:
No you're right, thanks for the catch! Fixing this in the next PR.